src/sadhana/IdenticalNotIdenticalArray.java

```diff
@@ -0,0 +1,33 @@
+package sadhana;
+
+//Coding exam_4
+
+public class IdenticalNotIdenticalArray {
+
+	void printData(String msg1, String msg2) {
+		boolean flag = false;
+		if (msg1.length() == msg2.length()) {
+			for (int i = 0; i < msg1.length(); i++) {
+				if (msg1.charAt(i) != msg2.charAt(i))
+					flag = true;
```

Comment:
same as above comment, need to put break, as you know for this char value's are not matching mean string are not identical, so we can update flag value and break the loop